### ISSUE TITLE
Thumbnail track support (DASH)

### ISSUE THREAD

**Issue by avelad** [google/shaka-packager#459]
Thumbnail in to DASH IF IOP 4.1 (see section 6.2.6 in http://dashif.org/wp-content/uploads/2017/09/DASH-IF-IOP-v4.1-clean.pdf)

Streams:
http://dash.edgesuite.net/akamai/bbb_30fps/bbb_with_tiled_thumbnails.mpd
http://dash.edgesuite.net/akamai/bbb_30fps/bbb_with_4_tiles_thumbnails.mpd
http://dash.edgesuite.net/akamai/bbb_30fps/bbb_with_tiled_thumbnails_2.mpd
http://dash.edgesuite.net/akamai/bbb_30fps/bbb_with_multiple_tiled_thumbnails.mpd
http://vm2.dashif.org/livesim-dev/testpic_2s/Manifest_thumbs.mpd

Is possible support it in Shaka Packager?

**Comment by kqyang** [comment]
Yes, we'd like to support it in 2019. Added to "2019 Planning" milestone.

**Comment by sr1990** [comment]
Hey @kqyang, any pointers for this one? Thanks.

**Comment by riba101** [comment]
It's been a while but is there a update on this feature? Would be very nice if this can be implemented.
